lib/elixir/lib/io.ex

```diff
@@ -230,9 +238,9 @@ defmodule IO do
   """
   @spec getn(chardata | String.Chars.t, pos_integer) :: chardata | nodata
   @spec getn(device, chardata | String.Chars.t) :: chardata | nodata
-  def getn(prompt, count \\ 1)
+  def getn(device, count \\ 1)
 
-  def getn(prompt, count) when is_integer(count) do
+  def getn(prompt, count) when is_integer(count) and count > 0 do
```

Comment:
I thought of that too while reviewing the code, and by doing that we can get rid of the body less function as well..
but I think whoever wrote that line is trying to make it clear that prompt is reading from stdin,
or that's what I understand from it.